src/xcode/ENA/ENAUITests/DeltaOnboarding/ENAUITests_06_DeltaOnboarding.swift

```diff
@@ -139,26 +139,4 @@ class ENAUITests_06_DeltaOnboarding: CWATestCase {
 		// leave screen
 		app.buttons[AccessibilityIdentifiers.NotificationSettings.close].waitAndTap()
 	}
-	
-	private func checkNotificationReworkScreen() {
```

Comment:
If you delete the hole test, the delta onboarding ui tests won't work.
Just remove lines 149 - 159 and replace them by the new alert?